Makefile

```
@@ -1,12 +1,15 @@
 rand := $(shell openssl rand -hex 6)
+ROOT_DIR := $(dir $(realpath $(lastword $(MAKEFILE_LIST))))
+DOCKER_UPSTREAM ?= $(shell cat $(ROOT_DIR)/.docker 2> /dev/null)
+DOCKER ?= $(shell which podman || which docker)
```

```suggestion
DOCKER_UPSTREAM := oryd/kratos-selfservice-ui-node
DOCKER := docker
```
Delete `.docker` dir and mention in the readme to use
```
make DOCKER=podman DOCKER_UPSTREAM=gcr.io/my/kratos
```